umm-lib/src/cmr/umm/acl_matchers.clj

```diff
@@ -37,22 +38,26 @@
       ;; Is end2 in the range
       (or (= end1 end2) (= start1 end2) (t/within? interval1 end2)))))
 
+(defn- parse-date
```

Comment:
I feel like we would have a common-util function for this already. If not, this function would probably go very nicely there

Reply:
I took it from the other acl_matchers.clj